internal/service/loadbalancer/manual_certificate_bundle.go

```diff
@@ -5,157 +5,219 @@ import (
-				ValidateDiagFunc: validateNameDiagFunc,
+var (
+	_ resource.Resource                = &manualCertificateBundleResource{}
+	_ resource.ResourceWithConfigure   = &manualCertificateBundleResource{}
+	_ resource.ResourceWithImportState = &manualCertificateBundleResource{}
+)
+
+func NewManualCertificateBundleResource() resource.Resource {
+	return &manualCertificateBundleResource{}
+}
+
+type manualCertificateBundleResource struct {
+	client *service.Service
+}
+
+func (r *manualCertificateBundleResource) Metadata(_ context.Context, req resource.MetadataRequest, resp *resource.MetadataResponse) {
+	resp.TypeName = req.ProviderTypeName + "_loadbalancer_manual_certificate_bundle"
+}
+
+// Configure adds the provider configured client to the resource.
+func (r *manualCertificateBundleResource) Configure(_ context.Context, req resource.ConfigureRequest, resp *resource.ConfigureResponse) {
+	r.client, resp.Diagnostics = utils.GetClientFromProviderData(req.ProviderData)
+}
+
+type manualCertificateBundleModel struct {
+	Certificate      types.String `tfsdk:"certificate"`
+	ID               types.String `tfsdk:"id"`
+	Intermediates    types.String `tfsdk:"intermediates"`
+	Name             types.String `tfsdk:"name"`
+	NotAfter         types.String `tfsdk:"not_after"`
+	NotBefore        types.String `tfsdk:"not_before"`
+	OperationalState types.String `tfsdk:"operational_state"`
+	PrivateKey       types.String `tfsdk:"private_key"`
+}
+
+func (r *manualCertificateBundleResource) Schema(_ context.Context, _ resource.SchemaRequest, resp *resource.SchemaResponse) {
+	resp.Schema = schema.Schema{
+		MarkdownDescription: "This resource represents manual certificate bundle",
+		Attributes: map[string]schema.Attribute{
+			"certificate": schema.StringAttribute{
+				MarkdownDescription: "Certificate within base64 string must be in PEM format.",
+				Required:            true,
 			},
-			"certificate": {
-				Description: "Certificate within base64 string must be in PEM format.",
-				Type:        schema.TypeString,
-				Required:    true,
+			"id": schema.StringAttribute{
+				MarkdownDescription: "Computed ID of the resource.",
+				Computed:            true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.UseStateForUnknown(),
+				},
 			},
-			"intermediates": {
-				Description: "Intermediate certificates within base64 string must be in PEM format.",
-				Type:        schema.TypeString,
-				Optional:    true,
+			"intermediates": schema.StringAttribute{
+				MarkdownDescription: "Intermediate certificates within base64 string must be in PEM format.",
+				Optional:            true,
+				Computed:            true,
 			},
-			"private_key": {
-				Description: "Private key within base64 string must be in PEM format.",
-				Type:        schema.TypeString,
-				Required:    true,
-				Sensitive:   true,
+			"name": schema.StringAttribute{
+				MarkdownDescription: "The name of the bundle must be unique within customer account.",
```

Comment:
Could split these descriptions to make them more clear, but not sure if it makes sense to do that in API docs first. E.g.:

```suggestion
				MarkdownDescription: "The name of the certificate bundle. Must be unique within customer account.",
```

Reply:
These descriptions have now been split